pkg/satellite/companies.go

```diff
@@ -24,6 +24,16 @@ type Companies interface {
 	Update(ctx context.Context, company *Company) error
 }
 
+// CompanyInput holds data needed to create/update Company
+type CompanyInput struct {
```

Comment:
`NewCompany` or `CreateCompany` to make it clearer that it's for creating the company.

Reply:
I am planing to use this inputs for update as well. It may feel weird to update a company with NewCompany or CreateCompany. CompayData could do. What do you think?